zerver/webhooks/slack/doc.md

```diff
@@ -1,34 +1,70 @@
+   and add `&slack_app_token=BOT_OAUTH_TOKEN` to the end of the URL you've just
+   generated in step 3. Your webhook url should now look like this:
 
-    If you'd like to map Slack channels to different topics within the same
-    channel, add `&channels_map_to_topics=1` to the end of the URL. Note that
-    this should be used instead of specifying a topic in the URL. If a topic
-    is specified in the URL, then it will be prioritized over the channel to
-    topic mapping.
+        {{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream=123
+        &slack_app_token=xoxb-123123123-123123123-XXXXXXXXX
 
-    If you'd like to map Slack channels to different channels, add
-    `&channels_map_to_topics=0` to the end of the URL. Make sure you create
-    channels for all your public Slack channels *(see step 1)*, and that the
-    name of each channel is the same as the name of the Slack channel it maps
-    to. Note that in this case, the channel to channel mapping will be
-    prioritized over the channel specified in the URL.
+1. Go to the **Event Subscriptions** menu, toggle **Enable Events** on and
+   enter your webhook url in the **Request URL** field.
 
-1. Go to <https://my.slack.com/services/new/outgoing-webhook>
-   and click **Add Outgoing WebHooks integration**.
+1. Still in the same menu, scroll down to the **Subscribe to bot events**,
+   **Add Bot User Event** and select the event `message.channels`.
 
-1. Scroll down and configure the **Channel** and/or **Trigger Word(s)**
-   options as appropriate. Set **URL(s)** to the URL constructed above,
-   and click **Save Settings**.
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/slack/001.png)
+
+### Channel Mapping
+If you'd like to map Slack channels to different topics within the same
+channel, add `&channels_map_to_topics=1` to the end of the URL. Note that
+this should be used instead of specifying a topic in the URL. If a topic
+is specified in the URL, then it will be prioritized over the channel to
+topic mapping.
+
+If you'd like to map Slack channels to different channels, add
+`&channels_map_to_topics=0` to the end of the URL. Make sure you create
+channels for all your public Slack channels *(see step 1)*, and that the
+name of each channel is the same as the name of the Slack channel it maps
+to. Note that in this case, the channel to channel mapping will be
+prioritized over the channel specified in the URL.
+
+### Slack APIs used
+
+- https://api.slack.com/events/message.channels, this event is used with Event
+   API to get Slack messages.
+- https://api.slack.com/methods/users.info, this end point is used to get
+   the username of each sender. The basic payload only include user_id
+- https://api.slack.com/methods/conversations.info, this API is used to get
+   the channel name of each message. The basic payload only include channel_id
+
+[slack_app_link]: https://api.slack.com/apps
```

Comment:
This can be replace by something like how's it done for the GitHub integration and just leave the link of the [Events API](https://api.slack.com/apis/events-api) and the [Slack Apps](https://api.slack.com/apps) page.